subsys/usb/class/hid/core.c

```diff
@@ -149,11 +149,28 @@ static void hid_int_in(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
 	hid_device.ops->int_in_ready();
 }
 
+static void hid_int_out(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
+{
+	u8_t data[CONFIG_HID_INTERRUPT_EP_MPS];
+	int len = 0;
```

Comment:
Now it is better but we can move stack variable data[] inside {} so that it would not always take memory, something like:

```
...
if (hid_device.ops->int_out_read && ...) {
    u8_t data[CONFIG_HID_INTERRUPT_EP_MPS];
    usb_read(ep, data, sizeof(data), &len);
    ...
}

```